fabric-api-base/src/main/java/net/fabricmc/fabric/api/util/BlockType.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2020 Legacy Fabric
+ * Copyright (c) 2016 - 2020 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.util;
+
+import net.minecraft.block.Block;
+
+/**
+ * Block Type Constants.
+ *
+ * @see Block#getBlockType()
+ */
+public class BlockType {
```

Comment:
do we really need this?

Reply:
yes we do. random integers in `getBlockType` make no sense just as how random integers in nbt make no sense

Reply:
in later versions this is an enum hence fapi doesnt have it